### ISSUE TITLE
UnexpectedLiquibaseException when dropping database objects using the latest H2 version

### ISSUE THREAD

**Issue by 0x7d7b** [liquibase/liquibase#2278]
## Environment

**Liquibase Version**: 4.6.2

**Database Vendor & Version**: H2 version 2.0.202

## Description

We have to update the H2 version used in our project from 1.4.200 to 2.0.202 due to CVE-2021-23463.
Unfortunately liquibase does not support that H2 version, yet. And the H2 developers did not port that fix back to a 1.4.x version.

## Steps To Reproduce

Given a table with an auto generated ```ID``` column:

```sql
CREATE TABLE FOO (
    ID BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
    ...
)
```

Invoke ```liquibase.database.Database#dropDatabaseObjects()``` programatically.

## Actual Behavior

Liquibase crashes with the following exception:

```shell
java.lang.IllegalStateException: liquibase.exception.UnexpectedLiquibaseException: liquibase.exception.DatabaseException: Error executing SQL SELECT SEQUENCE_NAME FROM INFORMATION_SCHEMA.SEQUENCES WHERE SEQUENCE_SCHEMA = 'YOURSCHEMA' AND IS_GENERATED=FALSE: Column "IS_GENERATED" not found; SQL statement:
SELECT SEQUENCE_NAME FROM INFORMATION_SCHEMA.SEQUENCES WHERE SEQUENCE_SCHEMA = 'YOURSCHEMA' AND IS_GENERATED=FALSE [42122-202]
```

This is because the ```INFORMATION_SCHEMA.SEQUENCES``` view does not contain the ```IS_GENERATED``` column.

## Expected/Desired Behavior

I'd expect liquibase to properly drop all objects in the currently connected schema.

**Comment by kataggart** [comment]
Thanks for writing up your issue @0x7d7b . We are reviewing and will be sure to get back to you. 

By the way, if you haven't had a chance to drop by https://forum.liquibase.org/ -- stop by!

**Comment by joschi** [comment]
> By the way, if you haven't had a chance to drop by [forum.liquibase.org](https://forum.liquibase.org/) -- stop by!

Unfortunately searching for "H2" isn't possible there because the search term is too short. 😉 
<img width="537" alt="Search term H2 is too short" src="https://user-images.githubusercontent.com/43951/146681416-23758ceb-905d-4eee-90b1-86181163ff50.png">

**Comment by nvoxland** [comment]
I have a fix for this in #2285

**Comment by comru** [comment]
Will a fix for this issue(#2278) be included in liquibase 4.3?
